src/Open3D/Geometry/IntersectionTest.cpp

```diff
@@ -54,12 +54,21 @@ bool IntersectionTest::TriangleTriangle3d(const Eigen::Vector3d& p0,
                                           const Eigen::Vector3d& q0,
                                           const Eigen::Vector3d& q1,
                                           const Eigen::Vector3d& q2) {
-    return NoDivTriTriIsect(const_cast<double*>(p0.data()),
-                            const_cast<double*>(p1.data()),
-                            const_cast<double*>(p2.data()),
-                            const_cast<double*>(q0.data()),
-                            const_cast<double*>(q1.data()),
-                            const_cast<double*>(q2.data())) != 0;
+    const Eigen::Vector3d mu = (p0 + p1 + p2 + q0 + q1 + q2) / 6;
+    const Eigen::Vector3d sigma =
+            (((p0 - mu).array().square() + (p1 - mu).array().square() +
+              (p2 - mu).array().square() + (q0 - mu).array().square() +
+              (q1 - mu).array().square() + (q2 - mu).array().square()) /
+             5)
```

Comment:
just for the information, why did we choose the unbiased `5` instead of `6` here?

Reply:
No real reason, I guess for the purpose of normalization it does not matter too much?